frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -21,7 +21,12 @@ import RecordsList from "../../components/RecordsList";
 const PHI_WARNING = {
   title: "Warning",
   content:
-    "The information contained in this transmission from Memorial Sloan-Kettering Cancer Center is privileged, confidential and protected health information (PHI) and it is protected from disclosure under applicable law, including the Health Insurance Portability and Accountability Act of 1996, as amended (HIPAA). This transmission is intended for the sole use of approved individuals with permission and training to access this information and PHI. You are notified that your access to this transmission is logged. If you have received this transmission in error, please immediately delete this information and any attachments from any computer.",
+    "The information contained in this transmission from Memorial Sloan-Kettering Cancer Center is privileged," +
+    " confidential and protected health information (PHI) and it is protected from disclosure under applicable law," +
+    " including the Health Insurance Portability and Accountability Act of 1996, as amended (HIPAA). This" +
+    " transmission is intended for the sole use of approved individuals with permission and training to access this" +
+    " information and PHI. You are notified that your access to this transmission is logged. If you have received" +
+    " this transmission in error, please immediately delete this information and any attachments from any computer.",
```

Comment:
I broke this out into multiple lines for readability when writing code. The content is unchanged.